doc/quickstart.md

```diff
@@ -2,8 +2,12 @@
 
 ## Prerequisites
 
-1. Kubernetes or Openshift cluster running on bare metal nodes.
-2. Multus-cni is deployed as default CNI plugin, and there is a default CNI plugin (flannel, openshift-sdn etc.) available for Multus-cni.
+1. A [supported](https://docs.openshift.com/container-platform/4.2/networking/multiple_networks/configuring-sr-iov.html#supported-devices_configuring-sr-iov) SRIOV hardware on the cluster nodes.
+2. Kubernetes or Openshift cluster running on bare metal nodes.
+3. Multus-cni is deployed as default CNI plugin, and there is a default CNI plugin (flannel, openshift-sdn etc.) available for Multus-cni.
+
+> **Note:** As for unsupported SRIOV NICs, that is not guaranteed but might work as well.
+> For that to happen, one must disable the webhook which validates the NIC's model.
```

Comment:
Please document how to disable it.

https://docs.openshift.com/container-platform/4.4/networking/hardware_networks/configuring-sriov-operator.html#disable-enable-sr-iov-operator-admission-control-webhook_configuring-sriov-operator

Reply:
well if it is written there I will just provide the link...right?

Reply:
This operator should be for both Kubernetes and OpenShift. I think that stating how to disable it here on U/S documentation would make it more accessible and send better message.